tests/integration/security/util/reachability/context.go

```diff
@@ -47,11 +47,14 @@ type TestCase struct {
 	Include func(src echo.Instance, opts echo.CallOptions) bool
 
 	// Handler called when the given test is being run.
-	OnRun func(ctx framework.TestContext, src echo.Instance, opts echo.CallOptions)
+	OnRun func(t framework.TestContext, src echo.Instance, opts echo.CallOptions)
 
 	// Indicates whether the test should expect a successful response.
 	ExpectSuccess func(src echo.Instance, opts echo.CallOptions) bool
 
+	// Allows filtering the destinations we expect to reach (optional).
+	ExpectDestinations func(src echo.Instance, dest echo.Instances) echo.Instances
```

Comment:
This seems a duplicate of the `Include` predicate function. The calloptions.Target could be used to check whether to include or not?

Reply:
Include doesn't allow modifying the destinations. For example, I _do_ want to include `b` as a target, but I only expect instances in specific clusters to be reach.